### PR TITLE
feat:Versioning of the APIs of the Discovery Services

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description

Introduced versioning of the APIs of the Discovery Services

- Introduced versioning of the APIs of the Discovery Services
- First version of this API is 1.0.

### CODE REVIEW

**Review by tunacicek — APPROVED** [review]
LGTM

**Comment by shijinrajbosch** [comment]
> @shijinrajbosch Please adjust changelog.md with new version.

Thanks for the review comments @tunacicek , I have updated the changelog accordingly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.6-M1
```
Comment (agg3fe) [inline CHANGELOG.md:8]: please make the version as 0.2.6 only and remove '-M1' from it.

Comment (shijinrajbosch) [reply CHANGELOG.md]: Thanks for the review comment @agg3fe  . I have incorporated the review comment.

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**